specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json

```diff
@@ -46,6 +34,9 @@
               "$ref": "#/definitions/PatternCreateObject"
             },
             "description": "The Name and Pattern of the feature."
+          },
+          {
+            "$ref": "#/parameters/ApiKey"
```

Comment:
How will this effect the code?

It might not a good idea to hard code the authentication header in operations.
One thing happening is we are supporting multiple authentication methods. For example we are working on supporting AAD authentication.

Reply:
At first it was add in SecurityDefinitions but it didn't reflect in the documentation and  [only allows 
 azure_auth oauth2 based on validation documnetation ](https://github.com/Azure/azure-openapi-validator/blob/master/docs/security-definitions-structure-validation.md) is there any better way?

Reply:
About `SecurityDefinitions`:
This `SecurityDefinitions` is originally used in our APIM service. After we moved to this repo, we didn't remove `SecurityDefinitions` as some services were still using them. In fact, some of the codegens can pick them up (but not every one). 
As migration work is basically done, we can try remove `SecurityDefinitions`.

About auto-generated API References:
Putting `Ocp-Apim-Subscription-Key` in operations can solve the document issue but I believe will mess up the auto-generated SDKs.
Missing support of `SecurityDefinitions` is just one of the issues of API References. Most of teams put authentication related information in hand-written document, we can do the same in LUIS:
https://docs.microsoft.com/en-us/azure/cognitive-services/luis/

Reply:
@yangyuan The authentication header change has been reverted.